test/test_foreach.py

```diff
@@ -119,6 +119,52 @@ def test_add_scalar_with_different_tensor_dtypes(self, device):
                    torch.tensor([1], dtype=torch.long, device=device)]
         self.assertRaises(RuntimeError, lambda: torch._foreach_add(tensors, 1))
 
+    def test_add_list_error_cases(self, device):
+        tensors1 = []
+        tensors2 = []
+
+        # Empty lists
+        with self.assertRaises(RuntimeError):
```

Comment:
nit: prefer self.assertRaisesRegex so that we can check that the error message is sane and makes sense

Reply:
You can just use a part of the string as the regex. E.g., "self.assertRaisesRegex(RuntimeError, "There were no tensor arguments to this function")"